vic/vic_run/include/vic_def.h

```diff
@@ -362,8 +362,8 @@ typedef struct {
 
     // Soil Constraints
     double SOIL_RARC;  /**< Architectural resistance (s/m) of soil when computing soil evaporation via Penman-Monteith eqn */
-    double SOIL_RESID_MOIST;  /**< Default residual moisture content of soil colum */
-    double SOIL_SLAB_MOIST_FRACT;  /**< Volumetric moisture content (fraction of porosity) in the soil/rock below the bottom soil layer; this assumes that the soil below the bottom layer has the same texture as the bottom layer. */
+    double SOIL_RESID_MOIST;  /**< Default residual moisture content (fraction of porosity) of soil colum */
```

Comment:
if it's now in mm then it should not be `fraction of porosity` I think - also `column` --> `column`

Reply:
It's now in `mm/mm`, so it is a fraction.

`colum` is a typo, sure, but I didn't introduce that typo - it was already there. But I can fix it.

Reply:
Right - since we're updating the comments, might as well also fix typos.